starter-code/css/main.css

```diff
@@ -0,0 +1,86 @@
+body{
+	font-family: Lato;
+	font-weight: normal;
+	color: black;
+	padding-left: 50px;
+	border-left: rgba(0,0,0,.4) 5px solid;
+	margin: 0 25% 0 25%;
```

Comment:
You can shorthand this `margin: 0 25%` which will set both top and bottom to 0, and left and right to 25%.